tests/test_commands/test_utils.py

```diff
@@ -254,3 +256,29 @@ def test_estimate_max_n_folds_backtest_with_transforms(
     run_estimate_max_n_folds_backtest_test(
         pipeline=pipeline_with_transforms, context_size=context_size, ts=ts, stride=stride, expected=expected
     )
+
+
+@pytest.mark.parametrize(
+    "params,to_remove,expected",
+    (
+        ({"start_timestamp": "2021-09-10"}, ADDITIONAL_FORECAST_PARAMETERS, {}),
+        (
+            {"prediction_interval": True, "n_folds": 3, "start_timestamp": "2021-09-10"},
+            ADDITIONAL_FORECAST_PARAMETERS,
+            {"prediction_interval": True, "n_folds": 3},
+        ),
+        (
+            {"prediction_interval": True, "n_folds": 3, "quantiles": [0.025, 0.975]},
+            ADDITIONAL_FORECAST_PARAMETERS,
+            {"prediction_interval": True, "n_folds": 3, "quantiles": [0.025, 0.975]},
+        ),
+        (
+            {"prediction_interval": True, "estimate_n_folds": True, "start_timestamp": "2021-09-10"},
+            ADDITIONAL_FORECAST_PARAMETERS,
+            {"prediction_interval": True},
+        ),
+    ),
+)
+def test_remove_params(params, to_remove, expected):
```

Comment:
You have tested only `ADDITIONAL_FORECAST_PARAMETERS`. This function isn't limited only to this parameter.

Reply:
Added a couple of tests